code/tyler/django/tyler_django_lab-2/blog_proj/templates/index.html

```diff
@@ -0,0 +1,25 @@
+{%extends 'base.html' %} {% block content %}
+
+<h1>Hello World!</h1>
+
+<form action="{%url 'blog_app:write_post' %}" method="link">
+  <a href="http://127.0.0.1:8000/blog/create_post.html/">
+    {% csrf_token %}
+    <button>Make a post</button>
+  </a>
+</form>
+<br />
+<form action="{%url 'users_app:profile' %}" method="link">
+  <a href="http://127.0.0.1:8000/users/profile.html/">
+    {% csrf_token %}
+    <button>My Profile</button>
+  </a>
+</form>
```

Comment:
The form elements are unnecessary. The `a` tags should be enough. The `href` attribute will be calculated by Django like the form's `action` attribute